model/project_parser.go

```diff
@@ -432,8 +433,9 @@ func LoadProjectFromVersion(v *Version, identifier string, shouldSave bool) (*Pr
 	if err != nil {
 		return nil, errors.Wrap(err, "error loading project")
 	}
-	if shouldSave {
-		if err := UpdateVersionProject(v.Id, v.ConfigUpdateNumber, pp); err != nil {
+	// NOTE: when we flip UseParserProject we don't need to check if ParserProject nil, because we would only be here if it was nil
```

Comment:
nit: For something like this, make it a TODO, and include a ticket number. That way if someone is grepping the code base for TODO's, or if someone sees this, they can check to see if the ticket is closed, and therefore know it's safe to make the change.